src/pages/proposals/create/StepReview.vue

```diff
@@ -225,3 +228,10 @@ export default {
         @next="$emit('next')"
       )
 </template>
+
+<style lang="stylus" scoped>
+  .disabled
```

Comment:
we have this classes in every step, i think we should remove duplication by moving in into global glass with something like `.step-disabled`. However I think border radius should follow the `.rounded` class because i think radius is not changing in terms of being disabled

Reply:
@arsenijesavic the disable for steps is a bit specific so I don't think it needs to be moved globally, although it's a good idea for everything else